.github/workflows/build-glfw.yml

```diff
@@ -0,0 +1,149 @@
+name: Build GLFW
+
+on:
+  workflow_dispatch:
```

Comment:
a manual workflow that rebuilds glfw for all the platforms on CI
similiar to what we have for LuaJit